params/config.go

```diff
@@ -463,10 +482,12 @@ func (c *ChainConfig) String() string {
 		engine = c.Ethash
 	case c.Clique != nil:
 		engine = c.Clique
+	case c.Parlia != nil:
+		engine = c.Parlia
 	default:
 		engine = "unknown"
 	}
-	return fmt.Sprintf("{ChainID: %v Homestead: %v DAO: %v DAOSupport: %v EIP150: %v EIP155: %v EIP158: %v Byzantium: %v Constantinople: %v Petersburg: %v Istanbul: %v, Muir Glacier: %v, Berlin: %v, London: %v, Arrow Glacier: %v, Engine: %v}",
+	return fmt.Sprintf("{ChainID: %v Homestead: %v DAO: %v DAOSupport: %v EIP150: %v EIP155: %v EIP158: %v Byzantium: %v Constantinople: %v Petersburg: %v Istanbul: %v, Muir Glacier: %v, Ramanujan: %v, Berlin: %v, London: %v, Arrow Glacier: %v, Engine: %v}",
```

Comment:
Can this be re-worked so that it only prints the fork with non nil block number? So we do not display irrelevant fork numbers on main net and BSC?

Reply:
Should I just leave this out then until the other changes for the other PRs (CLI and rest of BSC changes) are ready?

Reply:
Yes, you can leave config changes out, and bring it in separately, with re-work of this printout